### ISSUE TITLE
Code breaking bug: React Router Syntax outaded

### ISSUE THREAD

**Issue by pixelino** [jaredpalmer/razzle#1775]
### Current Behavior

When creating a new razzle appliction, starting the server with `yarn start` and navigating to `http://localhost:3000/` in the browser, this error is shown in the command line: 

```
Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check your code at server.js:26.

Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

  23 | export const renderApp = (req, res) => {
  24 |   const context = {};
> 25 |   const markup = renderToString(
  26 |     <StaticRouter context={context} location={req.url}>
  27 |       <App />
  28 |     </StaticRouter>

    at ReactDOMServerRenderer.render (/home/lino/code/my-app/node_modules/react-dom/cjs/react-dom-server.node.development.js:4053:17)
    at ReactDOMServerRenderer.read (/home/lino/code/my-app/node_modules/react-dom/cjs/react-dom-server.node.development.js:3690:29)
    at renderToString (/home/lino/code/my-app/node_modules/react-dom/cjs/react-dom-server.node.development.js:4298:27)
    at renderApp (/home/lino/code/my-app/src/server.js:25:1)
    at /home/lino/code/my-app/build/webpack:/src/server.js:52:1
    at Layer.handle [as handle_request] (/home/lino/code/my-app/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/lino/code/my-app/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/lino/code/my-app/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/lino/code/my-app/node_modules/express/lib/router/layer.js:95:5)
    at /home/lino/code/my-app/node_modules/express/lib/router/index.js:281:22

Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
```

### Expected behavior

Everything should work like expected.

### Reproducible example

Do the following:
1. `npx create-razzle-app my-app`
2. `cd my-app`
3. `yarn start`
4. navigate to `http://localhost:3000` in your browser

# Suggested solution(s)

To fix this behaviour I had to override following template code:

#### 1. src/server.js:3
`import { StaticRouter } from 'react-router-dom';`
to:
`import { StaticRouter } from 'react-router-dom/server';`

#### 2. src/App.js

```import React from 'react';
import { Route, Switch } from 'react-router-dom';
import Home from './Home';
import './App.css';

const App = () => (
  <Switch>
    <Route exact path="/" component={Home} />
  </Switch>
);

export default App;
```
to:
```
import React from 'react';
import { Route, Routes } from 'react-router-dom';
import Home from './Home';
import './App.css';

const App = () => (
  <Routes>
    <Route path="/" element={<Home/>} />
  </Routes>
);

export default App;

```

### Additional context

/

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Razzle           | 4.2.7
| Razzle Plugins   | ?
| Node             | v16.13.1
| Browser          | Firefox
| npm/Yarn         | yarn v1.22.17
| Operating System | Ubuntu WSL on Windows 10
| TypeScript       | /
| React            | 17.0.2

**Comment by fivethreeo** [comment]
Could you do a pull request?

https://github.com/jaredpalmer/razzle/blob/master/packages/create-razzle-app/templates/default/src/App.js

**Comment by pixelino** [comment]
#1776

**Comment by fivethreeo** [comment]
Released v4.2.8
